tools/run_tests/performance/scenario_config.py

```diff
@@ -164,7 +172,7 @@ def scenarios(self):
           'benchmark_seconds': BENCHMARK_SECONDS
       }
       yield {
-          'name': 'protobuf_async_qps_unconstrained_%s'
+          'name': 'cpp_generic_async_qps_unconstrained_%s'
```

Comment:
Then I think it's the scenario that should be changed. Because otherwise this is redundant with the other cpp_generic_async_qps_unconstrained case above, isn't it?